### ISSUE TITLE
Unable to start game

### ISSUE THREAD

**Issue by GORE84** [Pyrdacor/freeserf.net#70]
Trying launching the game error box pops up.

```
[audio:Abstract]Invalid or unsupported MOD format.

========== Stack Trace: ==========

   at Freeserf.Audio.MOD..ctor(Buffer data)
   at Freeserf.Renderer.OpenTK.Audio.Windows.WindowsModPlayer.CreateTrack(Int32 trackID)
   at Freeserf.Audio.Audio.Player.PlayTrack(Int32 trackID)
   at Freeserf.LocalSpectatorViewer.OnNewGame(Game game)
   at Freeserf.GameManager.SetCurrentGame(Game newGame)
   at Freeserf.GameManager.StartGame(GameInfo gameInfo, IRenderView renderView, IAudioInterface audioInterface)
   at Freeserf.UI.Interface.OpenGameInit(GameType gameType)
   at Freeserf.LocalPlayerViewer..ctor(IRenderView renderView, IAudioInterface audioInterface, Viewer previousViewer, Gui gui)
   at Freeserf.UI.Gui..ctor(IRenderView renderView, IAudioInterface audioInterface)
   at Freeserf.Renderer.OpenTK.GameView..ctor(DataSource dataSource, Size virtualScreenSize, DeviceType deviceType, SizingPolicy sizingPolicy, OrientationPolicy orientationPolicy)
   at Freeserf.FreeserfForm.FreeserfForm_Load(Object sender, EventArgs e)
```

I'm using client v1.9.23. OS: Windows 7

**Comment by Pyrdacor** [comment]
Which data files do you use? Amiga files?

**Comment by Pyrdacor** [comment]
For Amiga you need the following files:
gfxchip
gfxfast
gfxheader
gfxpics
music
sounds

For DOS you need SPAx.PA where x is the language shortcut.

If it does not work with Amiga files, try with DOS file.

**Comment by Pyrdacor** [comment]
Fixed it in v1.9.24.
